packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -415,7 +415,7 @@ export interface CallClientState {
   /**
    * Stores an ACS aquired phone number for making PSTN calls.
```

Comment:
Let's avoid using abbreviations like ACS.
Also I would recommend using a more verbose doc copied from the headless sdk
/**
     * A phone number in E.164 format that will be used to represent callers identity.
     * For example, using the alternateCallerId to add a participant using PSTN, this number will
     * be used as the caller id in the PSTN call.
     */